### PR TITLE
Classify what is an example and what isn't

### PR DESCRIPTION
**Pull request by pavpanchekha**
This PR marks some code blocks as examples that incorrectly weren't marked that way. It also marks some example tags as `.output` instead; this is an idea from #1161 that I thought made marking code blocks easier. All of the marks were cross-referenced with the printed book. I think of this both as superseding #1161 and also as preparation for trying to make the style of the online book look more like the printed book.

### CODE REVIEW

File: book/accessibility.md [hunk 1]
```diff
@@ -579,6 +579,7 @@ we also need to set dark mode when changing tabs, since all tabs should
 be either dark or light:
 
 ``` {.python}
+class Browser:
```
Comment (pavpanchekha) [inline book/accessibility.md:582]: This is I think an error in the proof that we *didn't* point out to the publisher. Honestly, it looks kind of fine in the book, not the end of the world, so it might not be worth pointing out anyway (it might repaginate). But we might as well correct it in the online version where it won't repaginate anything.

File: book/html.md [hunk 2]
```diff
@@ -514,16 +514,16 @@ def add_tag(self, tag):
...
+``` {.output}
+ <html>
+    <head>
+      <meta>
+      <link>
+      <link>
+      <link>
+      <link>
+      <link>
+      <meta>
```
Comment (pavpanchekha) [inline book/html.md:526]: Added a space because our printer does do that.

File: book/scripts.md [hunk 3]
```diff
@@ -60,11 +60,8 @@ the `duktape` library that `dukpy` uses.
...
 To test whether you installed DukPy correctly, execute this:
 
-``` {.python}
-import dukpy
-```
-
 ``` {.python .example}
+import dukpy
 dukpy.evaljs("2 + 2")
 ```
```
Comment (pavpanchekha) [inline book/scripts.md:66]: We ultimately decided to do this in the book.

File: src/runtime12.js [hunk 4]
```diff
@@ -93,10 +93,7 @@ function requestAnimationFrame(fn) {
 }
 
 function __runRAFHandlers() {
-    var handlers_copy = [];
-    for (var i = 0; i < RAF_LISTENERS.length; i++) {
-        handlers_copy.push(RAF_LISTENERS[i]);
-    }
+    var handlers_copy = RAF_LISTENERS;
```
Comment (chrishtr) [inline src/runtime12.js:96]: why this change?

File: infra/compare.py [hunk 5]
```diff
@@ -131,6 +131,7 @@ def compare_files(book, code, language, file):
                 print(f"{block.loc}:", error)
                 failure += 1
             continue
+        if "output" in block.classes: continue
```
Comment (chrishtr) [inline infra/compare.py:134]: don't we need a change to book.css?

File: book/accessibility.md [hunk 6]
```diff
@@ -1135,7 +1136,7 @@ If you print out `focusable_nodes` for the
 [focus example](examples/example14-focus.html), you should
 get this:
 
-``` {.python .example}
+``` {.python .output}
```
Comment (chrishtr) [inline book/accessibility.md:1139]: I don't fully understand the change from example to output. What impact does it have on the rendered book?

File: book/html.md [hunk 7]
```diff
@@ -514,16 +514,16 @@ def add_tag(self, tag):
...
-   <head>
-     <meta>
-     <link>
-     <link>
-     <link>
-     <link>
-     <link>
-     <meta>
+``` {.output}
+ <html>
```
Comment (chrishtr) [inline book/html.md:518]: why new spaces?

File: book/scheduling.md [hunk 8]
```diff
@@ -170,7 +170,7 @@ we'll implement `setTimeout` by saving the callback in a JavaScript
 variable and creating a handle by which the Python-side code can call
 it:
 
-``` {.javascript file=runtime}
```
Comment (chrishtr) [inline book/scheduling.md:173]: why is file= not needed anymore?

File: src/runtime12.js (hunk 4 above)
Comment (pavpanchekha) [reply src/runtime12.js:96]: Turns out the book had diverged from the source code and it wasn't being caught before.

Comment (pavpanchekha) [reply src/runtime12.js:96]: This moves the book code into the code code. I think the book code is nicer.

File: infra/compare.py (hunk 5 above)
Comment (pavpanchekha) [reply infra/compare.py:134]: Yes—I was planning to do all the style updates in a separate PR but I can do it all in this one if you'd like. Or just make the minimal change in this PR and restyle in another.

File: book/accessibility.md (hunk 6 above)
Comment (pavpanchekha) [reply book/accessibility.md:1139]: None, I found it helpful. `example` is code, `output` isn't code. If you don't like it I can make them all `example` but I sometimes find myself asking "is this really example code" and often that's because it's actually output.

File: book/html.md (hunk 7 above)
Comment (pavpanchekha) [reply book/html.md:518]: Our pretty-printer always prints a space between the indentation and the data.

File: book/scheduling.md (hunk 8 above)
Comment (pavpanchekha) [reply book/scheduling.md:173]: `runtime` is the default file.

Comment (pavpanchekha) [reply book/scheduling.md:173]: For JavaScript code.

File: infra/compare.py (hunk 5 above)
Comment (chrishtr) [reply infra/compare.py:134]: style in another PR is fine

File: src/runtime12.js (hunk 4 above)
Comment (chrishtr) [reply src/runtime12.js:96]: This new code is a pointer copy instead of a deep copy. Don't we need a deep copy?

Comment (pavpanchekha) [reply src/runtime12.js:96]: Ah great question. It's not a copy at all—it's a move, on the next line we reset `RAF_LISTENERS` to a newly-allocated vector. So what's important here is that `RAF_LISTENERS` and `handlers_copy` are disjoint, and the move is more convenient (and I guess faster, not that we care) than the deep copy & clear.

**Review by chrishtr — APPROVED** [review]
Please try out the rAF example from the examples directory for chapter 12 and make sure it works before landing this PR.

**Comment by pavpanchekha** [comment]
rAF examples fixed by #1346, put in pro-forma styling so we can merge this instead of leaving it hanging.
